src/bundle/Resources/public/scss/_tag.scss

```diff
@@ -15,7 +15,7 @@
 
     &__remove-btn {
         position: absolute;
-        top: calculateRem(7px);
+        top: calc(50% - 0.75rem);
```

Comment:
Note: `calculateRem` does not work inside `calc`.

Reply:
It works: `calc(50% - #{calculateRem(7px})`